test/salt/pillar/top.sls

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+base:
+  '*':
+    - defaults
+    - define_roles
```

Comment:
```suggestion
```

We've already got `define_roles` below using your "special hack".

Or perhaps we shouldn't confuse people who are using this formula as a template... and delete the two lines below instead.  What do you think?

Reply:
My fault, it's a typo.